docs/resources/Emoji.md

```diff
@@ -50,10 +50,10 @@
 ###### Gateway Reaction Custom Emoji Examples
 
 >info
->In `MESSAGE_REACTION_ADD` gateway events `animated` will be returned for animated emoji.
+>In `MESSAGE_REACTION_ADD` gateway events, `animated` will be returned for animated emoji.
 
 >info
->In `MESSAGE_REACTION_ADD` and `MESSAGE_REACTION_REMOVE` gateway events `name` may be `null` when custom emoji data is not available (for example, if it was deleted from the guild).
+>In `MESSAGE_REACTION_ADD` and `MESSAGE_REACTION_REMOVE` gateway events `name` maybe `null` when custom emoji data is not available (for example, if it was deleted from the guild).
```

Comment:
this is incorrect. "may be" (in other words, might be) is different from "maybe" (synonym of possibly), since "maybe" does not include a verb.